self-core-impl/src/main/java/com/selfxdsd/core/tasks/StoredTask.java

```diff
@@ -192,5 +192,10 @@ public String role() {
         public Invoices invoices() {
             return null;
         }
+
+        @Override
+        public Tasks tasks() {
+            return null;
+        }
```

Comment:
@criske Why do we need this method inside ``StoredTask``?

Reply:
that's from Unassigned Contract.

Reply:
@criske Right, forgot about that one...